static/src/stylesheets/module/onward-garnett/_rich-links-enhanced.scss

```diff
@@ -106,6 +106,20 @@
     }
 }
 
+// Opinion article specific styles
+//-----------------------
+
+.content--type-guardianview,
+.content--type-comment {
+    .rich-link {
+        background-color: $garnett-neutral-5;
+    }
+
+    .rich-link.tone-media--item {
```

Comment:
Why are we still using tones for media items? I thought one of the aims of the redesign is that we could start to get rid of tones?